ark/utils/example_dataset.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, dataset: str, overwrite_existing: bool = True, cache_dir: str
             "image_data": "image_data",
             "cell_table": "segmentation/cell_table",
             "deepcell_output": "segmentation/deepcell_output",
+            "example_pixel_output_dir": "segmentation/example_pixel_output_dir",
```

Comment:
The cell and pixel clustering should be in their own top-level folder, not contained within segmentation

Reply:
Got it